packages/shared/components/ActivityDetail.svelte

```diff
@@ -132,118 +132,200 @@
+            {#if date}
+                <div class="mb-5">
+                    <Text secondary>{localize('general.date')}</Text>
+                    <Text smaller>{date}</Text>
                 </div>
-                <Text smaller>{localize('general.you')}</Text>
-            {:else}
-                <Text smaller>{truncateString(senderAddress, 3, 3, 3) || localize('general.unknown')}</Text>
             {/if}
-        </div>
-        <Icon icon="small-chevron-right" classes="mx-4 text-gray-500 dark:text-white" />
-        <Text bold smaller>{formatUnitBestMatch(value, true, 2)}</Text>
-        <Icon icon="small-chevron-right" classes="mx-4 text-gray-500 dark:text-white" />
-        <div class="flex flex-col flex-wrap justify-center items-center text-center">
-            {#if receiverAccount}
-                <div
-                    style="--account-color: {receiverColor}"
-                    class="flex items-center justify-center w-8 h-8 rounded-xl p-2 mb-2 text-12 leading-100 font-bold account-color
-                    {isBright(receiverColor) ? 'text-gray-900' : 'text-white'}"
-                >
-                    {getInitials(receiverAccount.alias, 2)}
+            {#if senderAddress}
+                <div class="mb-5">
+                    <Text secondary>{localize('general.inputAddress')}</Text>
+                    <div class="flex flex-row justify-between items-center">
+                        <Text type="pre">{senderAddress}</Text>
+                        <CopyButton itemToCopy={senderAddress} />
+                    </div>
+                    <Text type="pre">
+                        {#if senderAccount}({senderAccount.alias}){/if}
+                    </Text>
                 </div>
-                <Text smaller>{localize('general.you')}</Text>
-            {:else}
-                {#each receiverAddresses as address}
-                    <Text smaller>{truncateString(address, 3, 3, 3) || localize('general.unknown')}</Text>
-                {/each}
             {/if}
+            {#if receiverAddresses.length > 0}
+                <div class="mb-5">
+                    <Text secondary>{localize('general.receiveAddress')}</Text>
+                    {#each receiverAddresses as receiver, idx}
+                        <div class="flex flex-row justify-between items-center">
+                            <Text type="pre">{receiver}</Text>
+                            <CopyButton itemToCopy={receiver} />
+                        </div>
+                        <Text type="pre" classes="mb-2 mt-0">
+                            {#if receiverAddressesYou[idx]}({receiverAddressesYou[idx].alias}){/if}
+                        </Text>
+                    {/each}
+                </div>
+            {/if}
+            <div class="mb-5 flex justify-center">
+                <button
+                    class="mobile-explorer-button action p-3 w-full text-center rounded-lg font-semibold text-14 bg-white text-blue-500"
```

Comment:
```suggestion
                    class="mobile-explorer-button action p-3 w-full text-center rounded-lg font-semibold text-14 bg-white dark:bg-gray-800 text-blue-500"
```
Maybe is lacking dark theme background? This change matches the light theme imo, what do you think?

Reply:
Yeah, it is, I alway forget about that, thank you 😅